app/uk/gov/hmrc/personaldetailsvalidation/connectors/ValidationIdValidator.scala

```diff
@@ -37,4 +37,11 @@ class ValidationIdValidator @Inject()(httpClient: HttpClient,connectorConfig: Co
     httpClient.GET[Boolean](url)
   }
 
+  private implicit val validationIdHttpReads: HttpReads[Boolean] = new HttpReads[Boolean] {
+    override def read(method: String, url: String, response: HttpResponse): Boolean = response.status match {
+      case OK => true
```

Comment:
I think this is good - we just need 3 cases, 2nd one is for NotFound (404) and catch all should throw a RuntimeException (unexpected response from backend)